portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,26 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Convert to JSON
+    String message1 = "My name is Micayla.";
+    String message2 = "I am from the Bay Area.";
+    String message3 = "I live in New York City.";
+    List<String> micaylaMessages = new ArrayList<>();
+    micaylaMessages.add(message1);
+    micaylaMessages.add(message2);
+    micaylaMessages.add(message3);
+
+    String json = convertToJsonUsingGson(micaylaMessages);
+
+    //Send response as JSON
```

Comment:
nit: for consistency and style, please add a space between the `//` and the `Send`. Also, please make comments complete sentences (or sentence fragments) with ending punctuation (here and above).